roles/nvidia-docker/tasks/main.yml

```diff
@@ -25,6 +25,14 @@
     mode: 0644
   register: daemon_json
 
+- name: grab nvidia-docker wrapper
```

Comment:
@dholt what was wrong with using the `nvidia-docker2` package? Why grab the wrapper directly?

Reply:
Oh man, I ran into a ton of version conflicts and dependency issues when you use the DGX role, kubernetes, want a newer docker version, etc. I can point you to the internal discussion.

Since `nvidia-docker2` is really just a tiny wrapper script and a meta package that creates a pain with versions, this seemed like a reasonable approach that avoids a lot of checking, dependency management and potential bugs.